### ISSUE TITLE
You deleted the Apache licence from the source you copied

### ISSUE THREAD

**Issue by ayasin** [llSourcell/The-Neural-Qubit#4]
Your removal of the licence comment at the top of `fraud_detection.py` (see commit https://github.com/llSourcell/The-Neural-Qubit/commit/5f89be146e36a0a34415c2b022e440e741e54b8a) is a violation of the Apache licence.  Please restore this to remain in compliance.

**Comment by ayasin** [comment]
The file appears to be copied from https://github.com/XanaduAI/quantum-neural-networks/blob/master/fraud_detection/fraud_detection.py

**Comment by Aspie96** [comment]
This is shameful.

**Comment by travel-code-sleep** [comment]
Yeah I don't think he understands what license means. Probably needs a "are you excited" video on licensing

**Comment by vinsis** [comment]
Is there a way we can report this or take some action? This is getting more and more blatant and shameless.

**Comment by furkanozbay** [comment]
What a shame!

**Comment by svaksha** [comment]
@ayasin / @vinsis, Github has a DCMA take down policy: https://help.github.com/en/articles/dmca-takedown-policy

**Comment by Aspie96** [comment]
I actually think it is possible this was a genuine mistake (since the other
files do contain the license, I see no reason to remove it from one, it
doesn't create the impression that he wrote the whole thing anyways).
Still, I hope he fixes this.

Il giorno gio 3 ott 2019 alle ore 20:41 ॥ स्वक्ष॥ svaksha <
notifications@github.com> ha scritto:

**Comment by svaksha** [comment]
While XanaduAI filed an issue to [restore the license](https://github.com/llSourcell/The-Neural-Qubit/issues/5) after they became aware of it, would you call it innocent/ ethical when its a repetitive pattern of taking work from multiple people/companies without respecting copyright, Foss licenses and attribution standards? Or only giving them credit when called out? 

* @gregwchase wrote [on reddit](https://www.reddit.com/r/MachineLearning/comments/d7ad2y/d_siraj_raval_potentially_exploiting_students/f260otg/) that his repo was plagiarised. He is right, MIT licenses are clear on not removing copyright. 
    * PS: The original author [was helping](https://github.com/gregwchase/dsi-capstone/issues/9) versus [no reply](https://github.com/llSourcell/AI_in_Medicine_Clinical_Imaging_Classification/issues/1). 


* @EmbersArc also had similar issues https://github.com/llSourcell/Landing-a-SpaceX-Falcon-Heavy-Rocket/issues/2#issuecomment-367670561 as does this issue: https://github.com/llSourcell/Landing-a-SpaceX-Falcon-Heavy-Rocket/issues/7

There are twitter and reddit threads with multiple people claiming fraud: https://www.reddit.com/r/MachineLearning/comments/d7ad2y/d_siraj_raval_potentially_exploiting_students/

**Comment by Aspie96** [comment]
I did not call it ethical.

Il giorno gio 3 ott 2019 alle ore 23:13 ॥ स्वक्ष॥ svaksha <
notifications@github.com> ha scritto:

**Comment by svaksha** [comment]
Glad to hear that :+1: as these license violations are not a `genuine mistake`.

**Comment by sajid-619** [comment]
This guy needs to be roasted, with great amount of spices

**Comment by raegen** [comment]
@Aspie96 taking into account the past week’s emergence of other examples that show just how humongous a pile of shit his pile of shit has in its basement, my best guess would be he either:
a) was too lazy to remove it from the files not visible in his videos, and he doesn’t even know enough to automate it.
b) he didn’t ever even open any of the other files.
On a serious note: I sincerely admire the amount of restraint you showed in judging this bastard.
